docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -240,7 +312,85 @@ \section{Proof of Concept Demonstration Plan}
+  Final Project Demo (Revision 1) & Full system demo (backend, frontend, datasets, etc.) & 24-Mar-25 & 30-Mar-25 & Final Demonstration (Revision 1) & 20\% & 7 & Coordinating all components for a seamless demo \\ \hline
+  EXPO Preparation & Prepare for Expo, polishing the UI, final system tweaks & 01-Apr-25 & Apr TBD, 2025 & EXPO Demonstration & 10\% & 15 & Finalizing and polishing all aspects of the project \\ \hline
+  Final Documentation & Compile final reports, code, guides (Problem Statement to Final Report) & 01-Apr-25 & 15-Apr-25 & Final Submission & 30\% & 15 & Completing all documentation to a high standard \\ \hline
+  \end{tabular}
+  \end{adjustbox}
+  \caption{Project Schedule Overview}
+  \end{table}
+
+
+  \section{Expected Technology}
+
+  \section{Frontend:}
+  \begin{itemize}
+      \item JavaScript: The primary language for the frontend development. It will handle user interactions and dynamic content updates on the web application.
+  
+      \item React: A popular JavaScript library used to build the user interface (UI). It will manage the frontend components and allow for a dynamic, responsive user experience.
+  \end{itemize}
+  
+  \section{Backend:}
+  \begin{itemize}
+      \item Node.js: Used for the server-side development, Node.js provides an asynchronous event-driven architecture, making it suitable for handling multiple client requests efficiently.
+      \item Python: Plays a major role in machine learning (ML) tasks, image processing, and backend scripting.
+      \item Flask: A lightweight Python framework used to build the backend server. It will handle API requests, manage server routing, and integrate with the database.
+  \end{itemize}
+  
+  \section{Machine Learning \& Data Processing:}
+  \begin{itemize}
+      \item TensorFlow (TF): The main machine learning framework to train or fine-tune models for tasks such as image classification and segmentation. We may also leverage pre-trained models, like Meta’s Segment Anything Model for segmentation tasks.
+      \item Numpy: A core library for numerical computing in Python, essential for handling arrays and performing mathematical operations efficiently.
+      \item PILLOW: A Python library used for image processing, enabling tasks like image resizing, format conversion, and basic manipulations.
+  \end{itemize}
+  
+  \section{Database:}
+  \begin{itemize}
+      \item SQLAlchemy: A powerful ORM (Object Relational Mapper) for Python that will facilitate database connections and work with both SQL and NoSQL databases.
+      \item SQL (PostgreSQL): Suitable for handling structured data and relationships. Ideal if we require ACID compliance and structured querying.
+      \item NoSQL (MongoDB): More flexible for handling unstructured data, allowing for greater scalability and schema-less design.
```

Comment:
I think we should just pick one (SQL/NoSQL) for now. We can always change as the project progresses, but as its written now it seems as though we'll use both

Reply:
Yes, I agree. For now I have listed Postgis as our preferred SQL db but we can change in future based on our needs.